client/src/components/graph/graph.js

```diff
@@ -366,6 +365,29 @@ class Graph extends React.PureComponent {
       );
     }
 
+    const svg = stateChanges.centroids || centroids;
```

Comment:
Where is this state set?  Or is this dead code?

Reply:
dead code, removed